### ISSUE TITLE
Experiment: repeat downweighting

### ISSUE THREAD

**Issue by gonzalobenegas** [Open-Athena/bolinas-dna#9]
## Description

Repetitive elements are a major component of eukaryotic genomes and are often interspersed with functional elements:
<img width="600" height="241" alt="Image" src="https://github.com/user-attachments/assets/a49da5fc-516f-4b98-bb0e-c52ec6a466da" />

Several papers have modified the training loss to downweight repeats, e.g. GPN family, PCAD family, Evo 2. It would be important to better understand the downweighting hyperparameter, and if there are alternatives (e.g. doing random shuffling of repeats with certain probability, similar in spirit to the data augmentation in GPN-MSA).

## Hypothesis or Goal

Downweighting repeats will generally improve downstream task performance, but the optimal weight might depend on species and/or repeat family.

### Links

[Wandb report](https://api.wandb.ai/links/gonzalobenegas/7lt9d69c)

## Results

Promoters, VEP task:
- Downweighting repeats improves downstream task performance and also seems to stabilize training.

**Comment by gonzalobenegas** [comment]
Closing for now in favor of more granular experiments.
